lib/ecto/query/api.ex

```diff
@@ -312,7 +312,28 @@ defmodule Ecto.Query.API do
   Only an interpolated string (binary) is supported, any other value will
   result into a compilation error or ArgumentError during runtime.
   """
-  def unsafe_fragment(fragment), do: doc! [fragment]
+  def unsafe_fragment(string), do: doc! [string]
+
+  @doc """
+  Sends an unsafe fragment directly to the database.
+
+  WARNING: sending an unsafe fragment may expose your application to sql
+  injection attacks. Think twice before using it and try to reach the same
+  result using the safe fragment/1 API.
+
+  This function will replace any `?` characters in `string` with values
+  from the `fragments` list.
+
+      unsafe_fragment(^"ROW_NUMBER() OVER(ORDER BY ?)", [sort])
```

Comment:
This example query won't work since a column cannot be a query parameter in SQL.